openid-4-verifiable-presentations-1_0.md

```diff
@@ -1611,17 +1611,30 @@ The signed request allows the Wallet to authenticate the Verifier using a trust
 
 ## Response
 
-Every OpenID4VP Authorization Request MUST result in a response being provided through the W3C Digital Credentials API. The Authorization Response is a JavaScript object, where the response parameters as defined for the Response Type are encoded as top-level members in this JavaScript object. 
+Every OpenID4VP Authorization Request MUST result in a response being provided through the W3C Digital Credentials API. This response is a JavaScript object as defined in [@!w3c.digital_credentials_api], where the OpenID4VP Authorization Response parameters as defined for the Response Type are encoded as members within the `data` member of this JavaScript object.
```

Comment:
This should probably just be a statement of fact, as the W3C spec already normatively mandates that the response is always  a `DigitalCredential`: 

```suggestion
Every OpenID4VP Authorization Request results in a response being provided through the W3C Digital Credentials API. The response is an instance of a `DigitalCredential` credential interface, as defined in [@!w3c.digital_credentials_api], and the OpenID4VP Authorization Response parameters as defined for the Response Type are represented as objects within the `data` attribute.
```

Reply:
Thanks Marcos!

On this part of the change:

> the OpenID4VP Authorization Response parameters as defined for the Response Type are represented **as objects** within the `data` attribute

Is 'represented as objects' technically correct given the values might be strings?

Reply:
Yeah, good point. What about if we only talk about the value of `.data`: "...Response Type is represented as an object within the `data` attribute" (updated above).

Reply:
Works for me - have changed an 'is' to 'are' as I think that reads better:

```suggestion
Every OpenID4VP Authorization Request results in a response being provided through the W3C Digital Credentials API. The response is an instance of a `DigitalCredential` credential interface, as defined in [@!w3c.digital_credentials_api], and the OpenID4VP Authorization Response parameters as defined for the Response Type are represented as an object within the `data` attribute.```